ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -117,6 +117,7 @@ patches:
           region: r1
           extraVol:
           - propagation:
+            - Manila
```

Comment:
Ok to propagate it unconditionally: if Manila isn't deployed, this propagation item doesn't have any effect.